src/util/manager/points.ts

```diff
@@ -83,6 +83,7 @@ export default class PointsManager extends Manager {
     let embed = new EmbedBuilder({
       description: `**${user} has approved \`${encodedData.activity}\` for <@${encodedData.snowflake}>!**\n[link to original message](${msg.url})`,
     }).setColor([0, 255, 0]);
+    if (msg.channel.isVoiceBased()) return;
```

Comment:
Here and a lot of other places where you add similar checks: if you are using `msg.channel`, do you still need to make sure it isnt a vc? You already know a message came from it, so surely it is a text channel..

Also i feel like I commented this above but I cant find the comment, apologies if this is a dupe

Reply:
Yeah, the discord API and discordjs updated to include all the different types of channels but for some reason it considers voice channels to be a TextBasedChannel. Might be because they did add those text chats as part of the voice channels. May look into that